public/js/controllers/HomeController.js

```diff
@@ -78,5 +78,36 @@ angular.module('BlocksApp').controller('HomeController', function($rootScope, $s
 
       }
   }
+})
+.directive('simpleSummaryStats', function($http) {
+  return {
+    restrict: 'E',
+    templateUrl: '/views/simple-summary-stats.html',
+    scope: true,
+    link: function(scope, elem, attrs){
+      scope.stats = {};
+
+      var statsURL = "/stats";
+      scope.stats.ethDiff = "...";

```

Comment:
oops!
this is a quick fix and have some garbage code...

Reply:
That's fine, another user is working on linking up all the stats.